ldapauthenticator/ldapauthenticator.py

```diff
@@ -150,6 +150,8 @@ def _validate_bind_dn_template(self, proposal):
         rv = []
         if isinstance(proposal.value, str):
             rv = [proposal.value]
+        elif isinstance(proposal.value, list):
```

Comment:
I think this should either be an unconditional else, or there should be a separate else clause that raises a TraitError.

Reply:
Yes, makes sense. I think if the value is neither `str` nor `list`, traitlets should raise an error anyways, right?! I mean we do not need to raise it manually I imagine. If so, I will change it to unconditional `else`.

Reply:
The allowed types are enforced by traitlets, list and strings are allowed